quickstart/latest/Fiddler_Quickstart_Model_Versions.ipynb

```diff
@@ -83,103 +88,112 @@
-    "PROJECT_NAME = 'model_version_sample_project' # UPDATE ME\n",
-    "DATASET_NAME = 'dataset_1' # UPDATE ME\n",
-    "MODEL_NAME = 'model_1' # UPDATE ME"
+    "try:\n",
+    "    fdl.init(url=URL, token=TOKEN) \n",
+    "except Exception as e:\n",
+    "    print(f'Error initializing Fiddler client: {e}')"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "6ad44441",
+   "id": "6e66776a",
    "metadata": {},
    "source": [
-    "Drop some output columns from the CSV file and pick the columns for inputs"
+    "#### 1.a Create New or Load Existing Project\n",
+    "\n",
+    "Once you connect, you can create a new project by specifying a unique project name in the fld.Project constructor and calling the `create()` method. If the project already exists, it will load it for use."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "80c407cf",
+   "id": "c968eae9",
    "metadata": {},
    "outputs": [],
    "source": [
-    "sample_df = pd.read_csv(DATASET_FILE_PATH)\n",
-    "column_list  = sample_df.columns\n",
-    "\n",
-    "input_columns  = list(column_list.drop([\"predicted_churn\",\"churn\", \"customer_id\", \"timestamp\"]))"
+    "try:\n",
+    "    # Create project\n",
+    "    project = fdl.Project(name=PROJECT_NAME).create()\n",
+    "    print(f'New project created with id = {project.id} and name = {project.name}')\n",
+    "except fdl.Conflict:\n",
+    "    # Get project by name\n",
+    "    project = fdl.Project.from_name(name=PROJECT_NAME)\n",
+    "    print(f'Loaded existing project with id = {project.id} and name = {project.name}')"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "6e66776a",
+   "id": "1390a569",
    "metadata": {},
    "source": [
-    "## 2. Create project"
+    "## 2. Load a Data Sample\n",
+    "\n",
+    "Load the sample dataset, store the list of columns, and create a subset of input columns (model features) for later use."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "c968eae9",
+   "id": "00ccde6a",
    "metadata": {},
    "outputs": [],
    "source": [
-    "try:\n",
-    "    # Create project\n",
-    "    project = fdl.Project(name=PROJECT_NAME).create()\n",
-    "    print(f'New project created with id = {project.id} and name = {project.name}')\n",
-    "except fdl.Conflict:\n",
-    "    # Get project by name\n",
-    "    project = fdl.Project.from_name(name=PROJECT_NAME)\n",
-    "    print(f'Loaded existing project with id = {project.id} and name = {project.name}')"
+    "sample_data_df = pd.read_csv(PATH_TO_SAMPLE_CSV)\n",
+    "column_list = sample_data_df.columns\n",
+    "input_columns = list(\n",
+    "    column_list.drop(['predicted_churn', 'churn', 'customer_id', 'timestamp'])\n",
+    ")\n",
+    "\n",
+    "sample_data_df"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "5bf916ca",
    "metadata": {},
    "source": [
-    "## 3. First version with no task\n",
+    "## 3. Add a Model\n",
```

Comment:
Create a model?  Onboard a model?  Are we consistent with this terminology?   The method's name is model.create().  I don't know if I feel strongly here other than we need to be consistent.